src/force.cpp

```diff
@@ -196,6 +196,28 @@ void Force::init()
   if (angle) angle->init();
   if (dihedral) dihedral->init();
   if (improper) improper->init();
+
+  // print warnings if topology and force field are inconsistent
+
+  if (comm->me == 0) {
+    if (!bond && (atom->nbonds > 0)) {
+      error->warning(FLERR,"Bonds are defined but no bond style is set");
+      if ((special_lj[1] != 1.0) || (special_coul[1] != 1.0))
+	error->warning(FLERR,"1-2 special neighbor interactions != 1.0");
```

Comment:
For these 3 additional warnings on special coeffs, I suggest starting them with Likewise.
E.g. Likewise 1-2 special ...
So it's more clear they are being issued for the same reason (bonds listed, no bond style)

Reply:
good idea. done.